package/Dockerfile

```diff
@@ -53,7 +53,7 @@ ENV LINODE_UI_DRIVER_VERSION v0.3.0
 # make sure the version number is consistent with the one at Line 100 of pkg/data/management/machinedriver_data.go
 ENV DOCKER_MACHINE_HARVESTER_VERSION v0.4.0
 ENV CATTLE_KDM_BRANCH ${CATTLE_KDM_BRANCH}
-ENV HELM_VERSION v3.8.0
+ENV HELM_VERSION v3.9.0
```

Comment:
Could we also update HELM_VERSION in https://github.com/rancher/rancher/blob/release/v2.6/Dockerfile.dapper#L25 and https://github.com/rancher/rancher/blob/release/v2.6/tests/validation/Dockerfile.v3api#L9